### PR TITLE
docs: Corrections and Additions to Airliner Flying Guide Existing Content

### PR DESCRIPTION
**Pull request by tshomas**
Weather.md - Correction of typos and formatting.
Navigation.md - Extension of VNAV paragraph.
Altitude-refs.md - Change of definitions and capitalisations.

Minor additions/changes without changing any major sections.

### CODE REVIEW

File: docs/airliner-flying-guide/navigation.md [hunk 1]
```diff
@@ -154,7 +154,12 @@ To know of the position of the aircraft, the Inertial Reference Systems (IRS) ar
...
+
+With a good database inside the MCDU, the Airbus family can fly an entire SID or STAR with no interaction from the pilots, if the data has been checked beforehand. Speed and altitude restrictions are followed to match the chart information.
+
+The Airbus Flight Management will try to calculate a 3-degree path backwards from the runway you are landing at up to the current cruising level. This will generate a point on the Navigation Display for the top of descent where if the altitude is set lower than the cruising level, and you push for DES, then the aircraft will follow the 3-degree path. The altitude always needs to be set lower than your current altitude, as the Airbus will stop at whatever is set in the FCU i.e., it will not carry on with the VNAV path. Unlike the Boeing fleet, the Airbus will also not automatically descend at ToD and requires a descent mode to be selected.
```
Comment (Valastiri) [inline docs/airliner-flying-guide/navigation.md]: ```The Airbus Flight Management will try to calculate a 3-degree path backwards from the runway you are landing at up to the current cruising level.```

The tail end of this sentence reads a bit weird. Specifically "the runway you are landing `at up to the` current crusing level"

File: docs/airliner-flying-guide/altitude-refs.md [hunk 2]
```diff
@@ -27,7 +27,7 @@ As you may fly from a country that uses one setting, to another country that use
 
 ***
 
-## Transition Altitudes and Transition levels
+## Transition Altitudes and Transition Levels
```
Comment (Valastiri) [inline docs/airliner-flying-guide/altitude-refs.md:30]: Small heads up for your future documenation, this is probably a good idea for the .md formatting in general, but mkdocs will render `##` / `H2` as all caps anyways. 

No issue here!

**Review by Valastiri — COMMENTED** [review]
Built locally and tested. No major issues.

File: docs/airliner-flying-guide/altitude-refs.md (hunk 2 above)
Comment (tshomas) [reply docs/airliner-flying-guide/altitude-refs.md:30]: Ah of course, maybe if I checked the final render first I'd have known that...just my anal side when I was going through the md!

File: docs/airliner-flying-guide/navigation.md (hunk 1 above)
Comment (tshomas) [reply docs/airliner-flying-guide/navigation.md]: Yeah it is a bit messy. how about:
"The Airbus Flight Management will try to calculate a 3-degree path from the start of the runway you are landing on, working backwards through your flight plan to achieve all the restrictions until it arrives at your current cruising altitude"

Comment (Valastiri) [reply docs/airliner-flying-guide/navigation.md]: lgtm!

**Comment by Valastiri** [comment]
Reviewed by Tarek. LGTM.

**Pull request merged**